internal/admin/dashboard/templates/sidebar.html

```
@@ -47,17 +47,20 @@ <h1>GoModel</h1>
     </nav>
     <div class="sidebar-footer">
         <div class="theme-toggle">
-            <button class="theme-btn" :class="{ active: theme === 'light' }" @click="setTheme('light')" title="Light" tabindex="-1">
+            <button class="theme-btn" :class="{ active: theme === 'light' }" @click="setTheme('light')" title="Light theme" aria-label="Light theme">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)